eurovision/notebooks/story.ipynb

```diff
@@ -1301,6 +1372,14 @@
     "df_pairs.head()"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To prepare the data for plotting, we filter for the top five country pairs that give each other high, low and unequal votes."
```

Comment:
I like the way you draw the reader through.

Reply:
Thanks!